source/Tests/Playnite.Tests/FileSystemTests.cs

```diff
@@ -68,5 +68,28 @@ public void CheckSumTests()
             StringAssert.AreEqualIgnoringCase("46fcb37aa8e69b4ead0d702fd459299d", FileSystem.GetMD5(testFile));
             StringAssert.AreEqualIgnoringCase("D8B22F5D", FileSystem.GetCRC32(testFile));
         }
+
+        [Test]
+        public void DirectorySizeScanTest()
+        {
+            using (var tempPath = TempDirectory.Create())
+            {
+                var filePath = Path.Combine(tempPath.TempPath, "DummyFile");
+                FileSystem.DeleteFile(filePath);

```

Comment:
Not needed since TempDirectory is created empty by default.

Reply:
Fixed in https://github.com/JosefNemec/Playnite/pull/3059/commits/8c18d0578183974b3ffd9339f33e9b4a167cc511